src/main/java/com/minecolonies/coremod/entity/mobs/aitasks/EntityAIAttackArcher.java

```diff
@@ -117,8 +123,15 @@ private void attack(final LivingEntity target)
 
             if (lastAttack <= 0 && entity.canEntityBeSeen(target))
             {
-                final ArrowEntity arrowEntity = EntityType.ARROW.create(target.world);
+                AbstractArrowEntity arrowEntity = EntityType.ARROW.create(target.world);
                 arrowEntity.setShooter(entity);
+
+                final ItemStack bow = entity.getHeldItem(Hand.MAIN_HAND);
+                if (bow.getItem() instanceof BowItem)
+                {
+                    arrowEntity = ((BowItem) bow.getItem()).customeArrow(arrowEntity);
```

Comment:
type, `custom` please =D

Reply:
this is the vanilla method name, this is not my personal choice =D

Reply:
that's annoying, guess MCP has a typo then lol